### ISSUE TITLE
Bars lose data when min/max of data exceeds breaks

### ISSUE THREAD

**Issue by davidhodge931** [davidhodge931/ggblanket#125]
If min limit = min of the data then x_limits[1] should be NA by default
If max limit = max of the data then x_limits[2] should be NA

```
library(ggblanket)
library(tidyverse)

storms %>% 
  group_by(year) %>% 
  summarise(across(wind, ~ mean(.x, na.rm = T))) %>% 
  filter(year >= 1980, year <= 2020) %>% 
  gg_col(x = year, 
         y = wind, 
         x_breaks = seq(1980, 2020, 10), 
         #x_limits = c(NA, NA)
         )
```

**Comment by davidhodge931** [comment]
For dates, should be NA_Date_

**Comment by davidhodge931** [comment]
```
  #before x_limits are calculated
  x_limits_null <- ifelse(rlang::is_null(x_limits), TRUE, FALSE)
  
  #after x_limits are calculated
  if (x_limits_null) {
    if (min(rlang::eval_tidy(x, data) == min(x_breaks)) {
      if (is.Date(rlang::eval_tidy(x, data))) {
        x_limits[1] <- lubridate::NA_Date_
      } 
      else if (is.numeric(rlang::eval_tidy(x, data))) {
        x_limits[1] <- NA
      }
    }
    else if (max(rlang::eval_tidy(x, data) == max(x_breaks)) {
      if (is.Date(rlang::eval_tidy(x, data))) {
        x_limits[2] <- lubridate::NA_Date_
      } 
      else if (is.numeric(rlang::eval_tidy(x, data))) {
        x_limits[2] <- NA
      }
    }
  }
```

**Comment by davidhodge931** [comment]
If I made this change...

Current:
Where someone specifies x_breaks and not x_limits, it defaults to making the plot with the x_limits min/max of x_breaks

If change made:
Where someone specifies x_breaks and not x_limits, it'd default to making the plot with the x_limits min/max of x_breaks, UNlESS the breaks do not cover the data - then it'd be NA.. 

Is this helpful or confusing??

**Comment by davidhodge931** [comment]
Leave as is

**Comment by davidhodge931** [comment]
The simplest way to deal with this without breaking everything is to change x_oob and y_oob defaults to scales::oob_squish

**Comment by davidhodge931** [comment]
New solution:

Keep oob_censor for everything, other than for x on bar, col, boxplot, crossbar, histogram, freqpoly

**Comment by davidhodge931** [comment]
histogram and freqpoly doesn't work as it bins all the squished values

**Comment by davidhodge931** [comment]
Possibly change col, boxplot and crossbar

**Comment by davidhodge931** [comment]
Solved by #134 
<img width="460" alt="image" src="https://user-images.githubusercontent.com/55065152/175128501-e4d620ca-e8d1-4cef-b2a7-1438fb30953d.png">
